README.md

```diff
@@ -21,15 +21,15 @@ go test
 
 ## About
 
-`retagger` is used to move all images required by Giant Swarm to our registry, while maintaining integrity of images - that is, we should be able to determine where all images came from. The alternative is manually retagging and pushing images, which means we lose a degree of accountability.
+`retagger` is used to move all images required by Giant Swarm to our Quay.io registry, while maintaining integrity of images - that is, we should be able to determine where all images came from. The alternative is manually retagging and pushing images, which means we lose a degree of accountability.
 
 A list of images (see `images.go`) is maintained, with a list of sha hashes and tag pairs. The `retagger` goes through each image, and then each sha hash and tag pair. The image is pulled by the sha hash, to overcome issues where a specific tag is rewritten, retagged with the tag, and pushed to our registry.
 
 The image name is also rewritten. For example, `quay.io/coreos/hyperkube` becomes `quay.io/giantswarm/hyperkube`, `prom/prometheus` becomes `quay.io/giantswarm/prometheus`.
 
-We attempt to pull the image from our registry first, to avoid unnecessary pulling of other images.
+We use the Quay.io API to check if an image has already been retagged, to avoid unnecessary work.
 
-We currently only support public images, both for pulling and pushing.
+We currently only support public images for pulling.
```

Comment:
Shouldn't it be `pushing`here? In the sense of pushing to our quay.io repo?

Reply:
This does feel ambiguous. I'll update.